Sources/PrimerSDK/Classes/Data Models/Apaya.swift

```diff
@@ -82,4 +82,91 @@ public struct Apaya {
             )
         }
     }
+    
+    class ViewModel {
```

Comment:
Are we sure we want this to be in the SDK too? I believe either API will return us the `carrierName` or the DEV has to do this.